src/tests/topp/FeatureFinderMetabo_2_noEPD_output.featureXML

```diff
@@ -527,7 +527,7 @@
 				<pt x="1.38960882" y="776.556250572984" />
 			</convexhull>
 			<UserParam type="string" name="label" value="T26_T5_T1"/>
-			<UserParam type="float" name="FWHM" value="43.5767288208008"/>
+			<UserParam type="float" name="FWHM" value="61.5109634399414"/>
```

Comment:
why does the FWHM increase here? Looking at the code, it should only decrease - is there something wrong with my thinking?

Reply:
It's possible that those are the edge cases. @timosachsenberg and me will take another look at it again next week.

Reply:
I dont see a a case where this could be correct, logically. If it cannot be correct then either the code is faulty or my reasoning is faulty. I am happy to be corrected.